### PR TITLE
Release 1.5

### PR DESCRIPTION
**Pull request by haijieg**
Update SDK to support GraphLab Create 1.5.

Break ABI compatibility to earlier versions.

### CODE REVIEW

**Comment by haijieg** [comment]
@ylow  Please review

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015, Dato, Inc.
+Copyright (c) 2015, Dato Inc.
```
Comment (znation) [inline LICENSE]: Why did you remove the comma on `Dato, Inc.`. Should this be `2015 Dato, Inc.`?

File: doxygen/Doxyfile [hunk 2]
```diff
@@ -32,7 +32,7 @@ DOXYFILE_ENCODING      = UTF-8
 # title of most generated pages and in a few other places.
 # The default value is: My Project.
 
-PROJECT_NAME           = "GraphLab Create 1.0 SDK"
+PROJECT_NAME           = "GraphLab Create 1.x SDK (BETA)"
```
Comment (znation) [inline doxygen/Doxyfile]: Should be 1.5?

**Comment by haijieg** [comment]
@znation Thanks, your feedback is addressed with the new commit.
